intel_extension_for_transformers/llm/runtime/graph/scripts/ci/cpp_graph_inference.sh

```diff
@@ -273,7 +273,14 @@ function main() {
     cd ..
 
     ## prepare example requiement
-    pip install -r "$requirements_file"
+    if [[ $requirements_file == *'.txt' ]]; then
+        pip install -r "$requirements_file"
+    elif [[ $requirements_file == *'.sh' ]]; then
+        source "$requirements_file"
```

Comment:
does ```source``` work in different kinds of shell?

Reply:
You are right. I just knew that source is only available in bash after some quick search. However, it should be fine in this case as we have `#!/bin/bash` in the beginning of `cpp_graph_inference.sh`

Reply:
> However, it should be fine in this case as we have #!/bin/bash in the beginning of 
Ok.